carapace-server/src/main/java/org/carapaceproxy/core/RuntimeServerConfiguration.java

```diff
@@ -252,6 +253,9 @@ public void configure(ConfigurationStore properties) throws ConfigurationNotVali
         // storing enabled for all peers by default
         localCertificatesStorePeersIds = properties.getValues("dynamiccertificatesmanager.localcertificates.peers.ids");
         LOG.log(Level.INFO, "dynamiccertificatesmanager.localcertificates.peers.ids={0}", localCertificatesStorePeersIds);
+
+        maxAttempts = properties.getInt("carapage.maxattempts", maxAttempts);
```

Comment:
This property is not dynamically updated if I change the maxattempts

Reply:
Typo: _carapa_**g**_e.maxattempts_

Reply:
Renamed to `dynamiccertificatesmanager.errors.maxattempts`